Sage/carbon
# chore(app-wrapper): remove classic theme

removal of classic theme from app-wrapper, delete old documentation and replaces stories with CSF
and MDX examples

fixes FE-3103

### Proposed behaviour
- Remove `classic theme` story
- Remove docgeninfo.json, README.md and notes.md files
- Update stories to be CSF and MDX formats

### Current behaviour
- `classic theme` story exists
- docgeninfo.json, README.md and notes.md files exist
- stories use the depreciated storiesOf API

### Checklist



### Additional context
N/A

### Testing instructions
- `git checkout FE-3103-remove_classic_theme_appwrapper`
- `npm start` to start Storybook
- Navigate to AppWrapper in Storybook
- No `classic` story should exist
- Documentation should be in an MDX format
